packages/flutter_tools/lib/src/isolated/devfs_web.dart

```diff
@@ -942,18 +942,12 @@ class WebDevFS implements DevFS {
     File metadataFile;
     List<String> modules;
     try {
-      final Directory parentDirectory =
```

Comment:
Are you reformatting 80 character lines to longer?  We're just fighting each other's autoformatting settings.  We should get on the same page so we're not thrashing.

Reply:
I don't generally mind if you reformat things to be shorter, but that has to be a reciprocal agreement

Reply:
You know I wouldn't have any character limits if I were Boss.  But I would rather just hit a button to format.  We can talk about it outside this PR, but I don't think you even touched this file other than formatting it?